# Generated for NaoQi method ALFrameManager.cleanBehaviors
# Stop playing any behavior in FrameManager, and delete all of them.

---


